operator_ui/src/connectors/redux/reducers/index.ts

```diff
@@ -14,6 +13,8 @@ import dashboardIndex, { IState as IDashboardState } from './dashboardIndex'
 import transactionsIndex from './transactionsIndex'
 
 export interface IState {
+  bridges: IBridgesState
```

Comment:
Shouldn't IState have all reducers annotated? Also I think you should go with `bridges?: IBridgesState`, because in test cases combineReducers won't pass all reducers needed and therefore will throw an error. Consequently, `combineReducers` on line 21 below, should be changed to `combinerReducers<IState>...`,

Reply:
Eventually when it is all converted they will be added to `IState`. This PR is meant to be changes specifically required to add valid annotations to `actions` + `api` and the knock on effects of these changes.